docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -116,7 +116,7 @@ regardless of which node it's (re)scheduled on.
 ### Ordinal Index
 
 For a StatefulSet with N replicas, each Pod in the StatefulSet will be
-assigned an integer ordinal, in the range [0,N], that is unique over the Set.
+assigned an integer ordinal, in the range [0,N), that is unique over the Set.
```

Comment:
In terms of correctness, I approve of this change. But at the risk of nit-picking your nit, may I suggest we go further and replace notation with words? That way the reader doesn't need to know any notation to understand.

For example:

> ... an integer ordinal, from 0 up through N-1, that is unique ...

What do you think?

Reply:
Sounds good to me.  Updated.